release-notes/release-notes-v10.10.1.md

```diff
@@ -34,6 +35,7 @@
 - Add `count_confirmed` parameter for get transactions count endpoints
 - Add `X-LEDGER-STATE` header in all API responses
 - Add `ledger_state` field in API v2 root endpoint
+- Remove `get_asset_names` and `get_asset_longnames` commands from API v1
```

Comment:
is this a breaking change?

Reply:
yes! these commands dump all the table assets without pagination!

Reply:
oh also they are both bugged on `master`.. since we have had no bug reports I guess no one uses them..